test/hotspot/jtreg/serviceability/jvmti/vthread/ThreadListStackTracesTest/ThreadListStackTracesTest.java

```diff
@@ -45,6 +45,13 @@ static void sleep(long millis) {
     }
 
     public void ensureReady(Thread vt, Thread.State expState) {
+        /*
+         * sleep for 1 second to allow the thread to reach the expected state,
+         * when the virtual thread is mounted, the virtual thread state,
+         * which is obtained from the carrier thread, can transition between
+         * WAITING and RUNNABLE before the reentrant lock parks the virtual thread
+         */
+        sleep(1000);
         // wait while the thread is not ready or thread state is unexpected
```

Comment:
The existing comment from upstream should have been left alone; the change does nothing but increase the likelihood of future merge conflicts. Same comment for https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/693.

Reply:
We will be more careful in the future. I can revert that line. Will reverting that line now prevent future merge conflicts?

Reply:
It depends how close the next change from upstream is. It may or may not conflict, but in general we should avoid unnecessary changes to code from upstream.

Reply:
Ack on the unnecessary changes. The risk for a merge conflict seems pretty low for that one-line. So, I feel that no preventive actions are needed.

Reply:
Agreed. It was just a suggestion aimed at avoiding unwanted differences relative to upstream. When reasonable, I think we should look at _any_ change we make to openjdk code as an opportunity to review *all* differences as compared with upstream.